survey-runner-queue/global_vars.tf

```diff
@@ -116,3 +116,7 @@ variable "logserver_cidr" {
 variable "audit_cidr" {
   description = "CIDR block of the centralised auditing service."
 }
+
+variable "delete_on_termination" {
+  description = "Delete EC2 volumes on termination of EC2 instance."
+}
```

Comment:
We should default this to false, as we want that to be the default production behaviour